islandora_simple_map.api.php

```diff
@@ -51,3 +51,19 @@ function hook_islandora_simple_map_parse_coordinates_callback() {
     ),
   );
 }
+
+
+/**
+ * Gather KML documents to render on a map.
+ *
+ * @param AbstractObject $object
+ *   The object for which the KML is to be gathered.
+ *
+ * @return array
+ *   An array of URLs pointing at publically-accessible KML documents.
+ */
+function hook_islandora_simple_map_gather_kml(AbstractObject $object) {
```

Comment:
Should this be `hook_islandora_simple_map_gather_geojson` as specified on line 563 of `includes/utilities.inc`??
```
function islandora_simple_map_gather_geojson(AbstractObject $object) {
  $geojson = module_invoke_all('islandora_simple_map_gather_geojson', $object);
  drupal_alter('islandora_simple_map_gather_geojson', $geojson, $object);
```

Reply:
honestly, I'd love to know that myself -- I was trying to drop the `module_invoke_all()` call since we're no longer calling a submodule, but I couldn't tell from the Drupal documentation how `drupal_alter()` was supposed to work at that point. Certainly having a hook function just straight up return an array containing a string didn't look right to me either...